docs/topics/braintree.rst

```diff
@@ -227,6 +227,16 @@ solitude.
 
     :status 201: payment method created.
 
+Change payment method on a subscription:
+
+.. http:post:: /braintree/subscription/change/
```

Comment:
``change`` is pretty vague if this is only allowing the payment method to be updated. I guess we might add more later? Otherwise ``/braintree/subscription/paymethod/`` seems fine to me.